prymer/util/executable_runner.py

```diff
@@ -30,6 +32,12 @@ class ExecutableRunner(AbstractContextManager):
     Subclasses of [`ExecutableRunner`][prymer.util.executable_runner.ExecutableRunner]
     provide additional type checking of inputs and orchestrate parsing output data from specific
     command-line tools.
+
+    Warning:
+        Users of this class must be acutely aware of deadlocks that can exist when manually
+        writing and reading to subprocess pipes. The Python documentation for subprocess and PIPE
+        has warnings to this effect as well as recommended workarounds and alternatives.
+        https://docs.python.org/3/library/subprocess.html
```

Comment:
This renders nicely:

<img width="788" alt="Screenshot 2024-10-17 at 3 34 43 PM" src="https://github.com/user-attachments/assets/127c995a-1aa9-4b4c-b807-52516ae8d4f7">